### ISSUE TITLE
Cocoapod can't install latest version 2.2.0

### ISSUE THREAD

**Issue by netgfx** [jinSasaki/InAppPurchase#41]
**Describe the bug**
Cocoapod can't install latest version 2.2.0

```
CocoaPods could not find compatible versions for pod "InAppPurchase":
  In Podfile:
    InAppPurchase (= 2.1.5)
```
or `2.2.0`

**To Reproduce**
1.   pod 'InAppPurchase', '2.1.5'
2. pod install

**Expected behavior**
Version 2.1.5 or 2.2.0 to be installed

**Smartphone (please complete the following information):**
 - Device: Any
 - OS: iOS 13
 - XCode: 11.1

**Comment by ericdesa** [comment]
The last published version on cocoapods is the `2.1.4` (https://github.com/CocoaPods/Specs/tree/master/Specs/e/c/9/InAppPurchase)

In the meantime, we can link the Podfile to the right git tag:
`pod 'InAppPurchase', :git => 'https://github.com/jinSasaki/InAppPurchase.git', :tag => '2.2.0'`

**Comment by jinSasaki** [comment]
@netgfx 
@ericdesa 

I'm sorry, I forgot to push the new version to Cocoapods.
I pushed InAppPurchase 2.2.0 just now! Please use the latest version.
